### PR TITLE
add English parameter description  for module: Plane

### PR DESCRIPTION
**Pull request by bofeng-song**

### CODE REVIEW

File: cocos/core/geometry/plane.ts [hunk 1]
```diff
@@ -47,10 +45,10 @@ export class Plane {
      * create a new plane
      * @zh
      * 创建一个新的 plane。
-     * @param nx 法向分量的 x 部分。
-     * @param ny 法向分量的 y 部分。
-     * @param nz 法向分量的 z 部分。
-     * @param d 与原点的距离。
+     * @param nx @en The x part of the normal component. @zh 法向分量的 x 部分。
```
Comment (minggo) [inline cocos/core/geometry/plane.ts]: The x component of normal vector.

File: cocos/core/geometry/plane.ts [hunk 2]
```diff
@@ -47,10 +45,10 @@ export class Plane {
...
+     * @param ny @en The y part of the normal component. @zh 法向分量的 y 部分。
+     * @param nz @en The z part of the normal component. @zh 法向分量的 z 部分。
+     * @param d  @en Distance from the origin. @zh 与原点的距离。
```
Comment (minggo) [inline cocos/core/geometry/plane.ts]: The distance between normal vector and the origin.

File: cocos/core/geometry/plane.ts [hunk 3]
```diff
@@ -62,8 +60,8 @@ export class Plane {
      * clone a new plane
      * @zh
      * 克隆一个新的 plane。
-     * @param p 克隆的来源。
-     * @return 克隆出的对象。
+     * @param p @en Source of cloning. @zh 克隆的来源。
```
Comment (minggo) [inline cocos/core/geometry/plane.ts]: The Plane object to be cloned from.

File: cocos/core/geometry/plane.ts [hunk 4]
```diff
@@ -90,11 +88,11 @@ export class Plane {
      * create a plane from three points
      * @zh
      * 用三个点创建一个平面。
-     * @param out 接受操作的对象。
-     * @param a 点 a。
-     * @param b 点 b。
-     * @param c 点 c。
-     * @return out 接受操作的对象。
+     * @param out @en The object to be operated on. @zh 接受操作的对象。
+     * @param a @en Points a. @zh 点 a。
```
Comment (minggo) [inline cocos/core/geometry/plane.ts]: Points -> Point

**Review by minggo — APPROVED** [review]

**Comment by pandamicro** [comment]
https://github.com/cocos/cocos-engine/issues/10840

**Pull request merged**